src/main/java/nextstep/subway/member/ui/MemberController.java

```diff
@@ -41,17 +43,23 @@ public ResponseEntity createMember(@RequestBody MemberRequest request) {
     }
 
     @GetMapping("/members/me")
-    public ResponseEntity<MemberResponse> findMemberOfMine() {
-        return ResponseEntity.ok().build();
+    public ResponseEntity<MemberResponse> findMemberOfMine(
+            @AuthenticationPrincipal LoginMember loginMember) {
+        MemberResponse member = memberService.findMember(loginMember.getId());
+        return ResponseEntity.ok().body(member);
     }
 
     @PutMapping("/members/me")
-    public ResponseEntity<MemberResponse> updateMemberOfMine() {
+    public ResponseEntity<MemberResponse> updateMemberOfMine(@AuthenticationPrincipal LoginMember loginMember) {
+        MemberRequest memberRequest
+                = new MemberRequest(loginMember.getEmail(), loginMember.getPassword(), loginMember.getAge());
```

Comment:
```suggestion
    public ResponseEntity<MemberResponse> updateMemberOfMine(@AuthenticationPrincipal LoginMember loginMember, MemberRequest memberRequest) {
```
해당 요청은 내 정보 수정을 처리하기 위한 컨트롤러 메서드입니다.
MemberRequest는 Request에서 받는게 좋을 것 같네요 :)

Reply:
아하 애초에 보내는 곳에서 MemberRequest로 보냈어야 하는군요